CONTRIBUTING.md

```diff
@@ -218,6 +249,8 @@ To edit the documentation, first clone the repository, and install `movement` in
 Then, install a few additional dependencies in your development environment to be able to build the documentation locally. To do this, run the following command from the root of the repository:
 ```sh
 pip install -r ./docs/requirements.txt
+# or, for uv users:
```

Comment:
https://github.com/neuroinformatics-unit/movement/blob/98f429e2badaf871741a562a6a1e1fc150c2f6ab/CONTRIBUTING.md?plain=1#L246 still says "first clone the repository". Should we also update this to match the "fork and clone" instruction in "creating a dev env" or simply update this line to: 
> To edit the documentation, ensure you have set up a [development environment](#creating-a-development-environment).